hibernate-core/src/main/java/org/hibernate/SharedSessionBuilder.java

```diff
@@ -67,6 +67,8 @@ public interface SharedSessionBuilder extends SessionBuilder {
 	 */
 	SharedSessionBuilder autoClose();
 
+	SharedSessionBuilder setTenantIdentifier(Object tenantIdentifier);
```

Comment:
@jrenaat after 8ceed5ab5a0d434086e8b3760fdf3fa9e5d168c4 we should not need this new method in `SharedSessionBuilder`.